resources/build.sh

```diff
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 rm -rf dist/ && mkdir -p dist/ &&
-babel src --ignore __tests__ --out-dir dist/ &&
+babel src --ignore __tests__ --out-dir dist/ --stage=0 &&
```

Comment:
Sure! Not sure how I missed the babel settings in package.json. I will add `es7.classProperties` explictly.